core/src/main/java/org/apache/accumulo/core/clientImpl/ConditionalWriterImpl.java

```diff
@@ -375,7 +376,7 @@ private TabletServerMutations<QCMutation> dequeue(String location) {
     this.auths = config.getAuthorizations();
     this.ve = new VisibilityEvaluator(config.getAuthorizations());
     this.threadPool = context.threadPools().createScheduledExecutorService(
-        config.getMaxWriteThreads(), this.getClass().getSimpleName());
```

Comment:
I didn't see any place where `this.getClass().getSimpleName` would change it's value so I switched it to a constant value. 

If there is different behavior here I'm happy to switch it back.

Reply:
I think it's fine based on my other comment.